src/Spice86.Core/Emulator/CPU/ModRM.cs

```diff
@@ -160,4 +170,59 @@ public class ModRM {
             _ => throw new InvalidModeException(_machine, _registerMemoryIndex)
         };
     }
+
+    private int ComputeOffset32(int mode, int rm) {
```

Comment:
Are you sure this is supposed to be signed?

Reply:
No. I'll change it. I can't really find it in the manual but it makes no sense for it to be negative since we cast it to ushort before using it anyway.